README.md

```diff
@@ -18,15 +18,50 @@ This repository is for the organization and governance of the Tessel project as
 
 For an introduction to the project's organization and key repos, check out the [Technical Overview](https://tessel.gitbooks.io/t2-docs/content/Debugging/Technical_Overview.html).
 
-## How can I get involved?
+# Current goals
 
-### Quick start
+On September 2, 2016, the Steering Committee set year-long goals. They are organized into two overarching goals, each with several supporting goals:
```

Comment:
Probably say "The Tessel Proejct has these goals for September 2017 (our next Tessel Summit)."

Also remove the overuse of "goals" in this paragraph.